mappings/net/minecraft/client/particle/Particle.mapping

```diff
@@ -46,7 +46,8 @@ CLASS net/minecraft/class_703 net/minecraft/client/particle/Particle
 		ARG 5 z
 	METHOD method_3064 getBoundingBox ()Lnet/minecraft/class_238;
 	METHOD method_3067 setBoundingBox (Lnet/minecraft/class_238;)V
-	METHOD method_3068 getColorMultiplier (F)I
+		ARG 1 boundingBox
+	METHOD method_3068 getBrightness (F)I
```

Comment:
Is this one correct?

Reply:
After looking at the code, it is indeed correct, it returns lightmap coordinates which does corresponds to the brightness.